docs/user/03-traces.md

```diff
@@ -24,33 +24,32 @@ For the recording of a distributed trace, every involved component must propagat
 
 In the Kyma cluster, the Telemetry module provides a central deployment of an [OTel Collector](https://opentelemetry.io/docs/collector/) acting as a gateway. The gateway exposes endpoints to which all Kyma modules and users’ applications should send the trace data to.
 
+The feature is optional, if you don't want to use the Traces feature, simply don't set up a TracePipeline.
```

Comment:
Is this fact part of the _architecture_ of the feature?

I gotta admit, I kind of miss the _"The Telemetry module provides the..."_ information in the Traces Overview section that we see in Application Logs.

Reply:
yes.. we would need it there to add this line..

Reply:
done